src/strategies/pob-hash-cv/index.ts

```diff
@@ -0,0 +1,55 @@
+import { subgraphRequest } from '../../utils';
+
+export const author = 'bonustrack';
```

Comment:
This should be your GitHub username
```suggestion
export const author = 'dave4506';
```